src/BlazorWasmSdk/Sdk/Sdk.targets

```diff
@@ -13,6 +13,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup>
     <_BlazorWebAssemblyTargetsFile Condition="'$(_BlazorWebAssemblyTargetsFile)' == ''">$(MSBuildThisFileDirectory)..\targets\Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets</_BlazorWebAssemblyTargetsFile>
+    <DisableImplicitNamespaceImports_BlazorWasm Condition="'$(DisableImplicitNamespaceImports_BlazorWasm)' == '' and '$(DisableImplicitNamespaceImports)' != 'true'">false</DisableImplicitNamespaceImports_BlazorWasm>
```

Comment:
`BlazorWebassemblyDisableImplicitNamespaceImports` if you expect this to be a `public` property. We don't use `_` anywhere afaik on our SDKs

Reply:
See https://github.com/dotnet/sdk/issues/18839

Reply:
Wouldn't it be nice if we had namespaces for MSBuild properties 😄